ilan/README.md

```diff
@@ -1,37 +1,21 @@
-# Building an IBM HTTP Server v8.5.5 ILAN image from binaries
+# Building an IBM HTTP Server ILAN image from binaries
 
-An IBM HTTP Server ILAN image can be built by obtaining the following binaries:
-* IBM Installation Manager binaries from [developerWorks](http://www.ibm.com/developerworks/downloads/ws/wasdevelopers/)
+An IBM HTTP Server ILAN image can be built by running one script.
 
-  IBM Installation Manager binaries:
-  * agent.installer.linux.gtk.x86_64_1.6.2000.20130301_2248.zip
+Versions available:
```

Comment:
Better not to include this here as it just gives us two things to update every time we add a new version. Better to just link them to versions.csv.